apps/ima/translations/src/en/fields.json

```diff
@@ -17,5 +17,114 @@
         "hint": "You cannot give immigration advice. You must answer all the questions as if you are the person named on the duty to remove letter."
       }
     }
+  },
+  "how-removal-condition-1-applies": {
+    "legend": " ",
```

Comment:
why is this blank?
you could just add a visuallyhidden CSS

Reply:
I've now included relevant text to all the blank legends and applied the `visuallyhidden` class name to them.

Reply:
👍